examples/src/examples/loaders/gsplat-many.example.mjs

```diff
@@ -70,11 +70,12 @@ assetListLoader.load(() => {
     camera.setLocalPosition(-3, 1, 2);
 
     // instantiate guitar with a custom shader
-    const guitar = assets.guitar.resource.instantiate({
-        vertex: files['shader.vert']
+    const guitar = new pc.Entity('guitar');
+    guitar.addComponent('gsplat', {
+        asset: assets.guitar
```

Comment:
we need to preserve the custom vertex shader in this example

Reply:
Yes! Followup PR.